apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -1331,6 +1331,16 @@ private func buildTheme() -> AppTheme {
     }
 }
 
+class ChatViewScrollState: ObservableObject, Equatable {
```

Comment:
What are the requirements here?
Could we use existing `ReverseListScrollModel` state to avoid creating a new `ObservableObject`?.
It already represents the scroll state.

Reply:
I changed to a new way based on your suggestion